README.md

```diff
@@ -40,6 +47,18 @@ snakeviz pycodcif_100.prof  # inspect one output
 * `pycifrw-fast_100.prof`: 17.6s spent in `ReadCif`
 * `pycodcif_100.prof`: 16.3s spent in `parse`
 
+### Ubuntu 18.04, Intel® Core™ i7-4790 CPU @ 3.60GHz × 8, HDD 
+* Note: Extended test set! * Heavy load (both memory and CPU) parallel to benchmark. Means and standard deviations from three runs (except for `pycifrw`, where we only did two runs).  
```

Comment:
Could you add a brief description at the end of "## Goal":

Test sets:

 * structures_100.zip: ...
 * structures_108.zip: ...